### PR TITLE
Add v3.account zh_CN localization

### PR DESCRIPTION
**Pull request by potterhe**

### CODE REVIEW

**Comment by jonkoops** [comment]
Hi @potterhe, could you get another native speaker to review this PR? Thanks!

**Comment by potterhe** [comment]
@xianli123 Can you help me view this PR?

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_zh_CN.properties [hunk 1]
```diff
@@ -1,49 +1,194 @@
...
+share=分享
+close=关闭
+email=Email
+signOutWarning=确认退出登陆？
+removeConsentError=由于以下原因无法删除同意\: {{error}}
+signOutAllDevicesWarning=此操作将登出所有已登入您帐号的设备，包括您目前使用的设备。
+unShareSuccess=资源已取消共享。
+signingIn=登陆凭证
+stopUsingCred=停止使用 {{name}}？
+groupsListHeader=群组列表头
```
Comment (xianli123) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_zh_CN.properties:63]:
```suggestion
groupsListHeader=群组列表表头
```

**Review by xianli123 — COMMENTED** [review]
Thanks @potterhe . I made a suggestion on line 63. Please take a look.  All the translations look good to me.

**Comment by potterhe** [comment]
Thanks @xianli123 , I have merged the suggestion.

**Review by xianli123 — APPROVED** [review]

**Comment by xianli123** [comment]
@potterhe My pleasure. It looks good. Thanks!

**Review by hmlnarik — APPROVED** [review]
Approving per @xianli123 's approval. Thank you for the contribution!

**Pull request merged**